src/api/controllers/conversations/conversations.list.js

```
@@ -0,0 +1,48 @@
+const errorHandler = require('../../utils/errorHandler')
+const isAuth = require('../../middlewares/isAuth')
+const Sequelize = require('sequelize')
+const log = require('../../utils/log')(module)
+
+module.exports = app => {
+  app.get('/conversations', [isAuth()])
+  app.get('/conversations', async (req, res) => {
+    const { Conversation, User, Team, Spotlight } = req.app.locals.models
+    const user = await User.findById(req.user.id)
+    if (user.isAdmin === 100) {
```

`if (user.isAdmin != 100) {error………;}` 
Fail fast, ça permet de gagner un niveau d'indentation

Ou carrément utiliser le middleware isAdmin ? ;)